.github/workflows/build.yml

```
@@ -92,15 +96,23 @@ jobs:
           load: true
           tags: ${{ steps.migrations-docker-metadata.outputs.tags }}
 
-      - name: "Run Tests"
-        run: yarn test
+      #- name: "Run Tests"
+      #  run: yarn test
+
+      - name: Authenticate to GCP
+        id: gcp-auth
+        uses: google-github-actions/auth@v1
```

actionlint (1.7.4)

the runner of "google-github-actions/auth@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)